File: Codeforces/p01a_Theatre_square/theatreSquare.ts
const input = require("prompt-sync")();

let [n, m, a] = input().split(" ").map(Number);

let side1: number = (n + a - 1) / a;
let side2: number = (m + a - 1) / a;

let result: number = side1 * side2;
console.log(result.toFixed(0));

File: Codeforces/p01a_Theatre_square/theatreSquare.kt
fun main(){
    val n: Long = readln().toLong()
    val m: Long = readln().toLong()
    val a: Long = readln().toLong()

    var side1: Long = (n + a - 1) / a
    var side2: Long = (m + a - 1) / a

    val result = side1 * side2
    println(result)
}